db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,22 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
```

Comment:
@detiber It describes the type of the deployment on the ManageIQ side, could be Managed-provision (we do provision on the nodes and then we run the ansible installer on them) Managed-existing (pick vms/instances that already exists on ManageIQ and run the installer) Unmanaged-exisiting ( existing vms that aren't managed)